### PR TITLE
Fix weird interaction in Exploration

### PR DESCRIPTION
**Pull request by DrDuckedGoose**
## About The Pull Request

Fixes a weird interaction with exploration & Psyphoza

I'd appreciate it if this could be TM'd to see if it targets the right issue. Although it does fix an issue on my machine, there may still be more affecting lower spec machines.

## Why It's Good For The Game

Bug fix

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
fix: Fix bug with exploration stacking filters
/:cl:

### CODE REVIEW

**Comment by EvilDragonfiend** [comment]
Is it safe?? not `list.Cut()`?

**Comment by Absolucy** [comment]
@EvilDragonfiend `filters` isn't an ordinary /list: https://github.com/BeeStation/BeeStation-Hornet/blob/master/code/game/atoms.dm#L1624-L1625

File: code/_onclick/hud/rendering/plane_master.dm [hunk 1]
```diff
@@ -262,6 +263,7 @@
 /atom/movable/screen/plane_master/anti_psychic/backdrop(mob/mymob)
 	. = ..()
 	//fixes issue with bloom outlines
+	remove_filter("hide_outline")
```
Comment (PowerfulBacon) [inline code/_onclick/hud/rendering/plane_master.dm:266]: This is unnecessary due to add_filter handling it

File: code/_onclick/hud/rendering/plane_master.dm [hunk 2]
```diff
@@ -246,6 +246,7 @@
 
 /atom/movable/screen/plane_master/psychic/backdrop(mob/mymob)
 	. = ..()
+	filters = null
 	filters += filter(type = "bloom", size = 2, threshold = rgb(85,85,85))
 	filters += filter(type = "alpha", render_source = "psychic_mask")
 	filters += filter(type = "radial_blur", size = 0.0125)
```
Comment (PowerfulBacon) [inline code/_onclick/hud/rendering/plane_master.dm:252]: This should use add_filter instead of filters += to avoid duplicates.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

**Comment by Crossedfall** [comment]
Superseded by #10334

**Pull request closed without merging**